circuit_maintenance_parser/parser.py

```diff
@@ -161,3 +161,30 @@ def clean_line(line):
             line = line.strip()
         # TODO: below may not be needed if we use `quopri.decodestring()` on the initial email file?
         return line.replace("=C2", "").replace("=A0", "").replace("\r", "").replace("=", "").replace("\n", "")
+
+
+class Csv(Parser):
+    """Csv parser."""
+
+    _data_types = ["application/csv", "text/csv", "application/octet-stream"]
+
+    def parse(self, raw: bytes) -> List[Dict]:
+        """Execute parsing."""
+        result = []
+
+        data_base: Dict[str, Union[int, str, Iterable]] = {}
+        try:
+            for data in self.parse_csv(raw, data_base):
+                result.append(data)
```

Comment:
even it's not necessary for this parser... I can foresee that maybe some provider could send mutliple maintenances on a CSV notification, so it's ok to leave as it is

Reply:
yes, I thought better to be safe for now.